src/pages/workspaces/workspace/analysis/notebook-utils.js

```diff
@@ -2,48 +2,23 @@ import _ from 'lodash/fp'
 import { Fragment, useState } from 'react'
```

Comment:
Is splitting parts of this off into tool-utils motivated by partially converting it to TypeScript or renaming the module? If the former, can we can go ahead and convert the whole file to avoid all the changes to import paths? If the latter, can we do reorganization in a separate PR to make it easier to see changes?